app/assets/javascripts/views/index.js

```diff
@@ -37,11 +37,7 @@ Yacs.views.index = function () {
               if (success)
                 Yacs.views.courses(content, data);
           });
-        } else {
```

Comment:
I think we should leave the 'pressing enter when blank bringing you back to departments page' behavior. I can't think of a case where this would be undesired.